packages/common/src/store/ui/modals/comment-confirmation-modal/commentConfirmationModal.ts

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from 'react'
+
+import { createModal } from '../createModal'
+
+export type CommentConfirmationModalState = {
+  messages: {
+    title: string
+    body: ReactNode
+    cancel: string
+    confirm: String
+  }
+  confirmButtonType?: 'primary' | 'destructive'
+  confirmCallback: () => void
+  cancelCallback?: () => void
+}
+
+const commentConfirmationModal = createModal<CommentConfirmationModalState>({
```

Comment:
Do we not have a generic ConfirmationModal? we have ConfirmationDrawer on modal

Reply:
Yeah in hindsight I was just following patterns here but it makes no sense to have so many `XYZConfirmationModal` lol. I can re-package this one as a generic `ConfirmationModal` for the future.
yet another reason to refactor modals 👀

Reply:
Update: Dylan added a generic `ConfirmationModal` in his PR that isn't redux-managed; so I updated to use that instead!